README.md

```diff
@@ -8,3 +8,40 @@ Run the container providing the spring profiles in CATALINA_OPTS
 
 docker run -p 8080:8080 -e CATALINA_OPTS="-Dspring.profiles.active=default,hsqldb" sdc-uaa
 
+# sdc-uaa endpoints
```

Comment:
Would be valuable to have an explanation of what each helper script does and how to use them. It's probably not worth listing the UAA endpoints as they're not part of this service and already well documented within their own documentation.

Reply:
What are the helper scripts, exactly? Are they the .sh files?

Reply:
I'd rename this title to `Helper scripts` but otherwise looking generally good